### ISSUE TITLE
Error no module named future

### ISSUE THREAD

**Issue by GatorByte** [pmorissette/bt#50]
The new version of ffn which bt uses requires the future module for compatibility between Python 2 & 3. If you used the Anaconda installation for Python and then install bt using "pip install bt" when you run bt you get an error no module named future. "future" is not in the standard Anaconda packages installed and not loaded when installing bt. Hope this isn't too trivial an issue but it took me a couple of hours to track down.
Using "conda install future" fixes the problem.

**Comment by pmorissette** [comment]
Hey @GatorByte,

Sorry to hear you lost some time on this. Let me see what I can do and I'll get back to you. 

Thanks for reporting the issue.

Cheers,
Phil

**Comment by GatorByte** [comment]
Phil,
Thanks for the quick reply but no need to worry about me, I installed 
future and everything is working for me. I already had bt running on
my desktop but wanted it on a new laptop using Windows 10 and a new 
download of Anaconda. I only wanted to point out for others that the 
current directions for installing bt by first installing Anaconda and 
then using pip install bt doesn't load one of the dependencies (future) 
which requires a separate "conda install future" step.
Again thank you very much for this software, I have been comparing 
different portfolio allocations for my personal retirement funds.
Rob

**Comment by pmorissette** [comment]
Hey @GatorByte,

Just wanted to let you know that this should now be fixed. I added future as a dependency in ffn, which bt builds upon. 

Thanks for letting me know,
Phil

**Comment by TC86** [comment]
Hi, I have tried to install ffn via `pip install ffn`, but I continually receive (very long) error messages. I have numpy, scipy, pandas, scikit-learn, all of which I was successfully able to install via `pip install`, plus I have the full Anaconda3 suite. If `pip install ffn` is not working for me, is there another way of installing ffn? If I downloaded the tar.gz directly, is there a manual way of installing ffn?
